custom_components/renault/binary_sensor.py

```diff
@@ -71,7 +77,8 @@ class RenaultChargingSensor(RenaultBatteryDataEntity, BinarySensorEntity):
     @property
     def is_on(self):
         """Return true if the binary sensor is on."""
-        return self.coordinator.data.get("chargingStatus") == 1
+        if self.data.chargingStatus is not None:  # Zero can be a valid value
+            return self.data.get_charging_status() == ChargeState.CHARGE_IN_PROGRESS
```

Comment:
if 0, should the binary be at on, off, undefined or unavailable state ?

Reply:
I think it should be undefined. I will explicit `None`